modules/api/src/test/java/com/intuit/wasabi/api/pagination/PaginationHelperTest.java

```diff
@@ -0,0 +1,175 @@
+/*******************************************************************************
+ * Copyright 2016 Intuit
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package com.intuit.wasabi.api.pagination;
```

Comment:
If I saw it correctly, the other test classes do not use the License Header. We should check if this is wanted or what the thought behind it is (I guess a test class is not worth protecting since it does not contain business logic).